api/src/main/resources/workbench-api.yaml

```diff
@@ -933,6 +933,55 @@ paths:
           description: A list of workspaces
           schema:
             "$ref": "#/definitions/WorkspaceListResponse"
+  "/v1/admin/audit/workspace":
```

Comment:
There could certainly be useful queries that involve both a user and a workspace (or various numbers of them). Though I don't know I've left room for that.

Reply:
Makes sense, but it sounds a bit like YAGNI, as I don't know of any concrete use cases that would lead us to include such a query in the API.

On the balance, I think aligning with the existing collection hierarchy will be clearer and more consistent with existing API paths.

Reply:
That's fine I guess. I didn't want to be married to the namespace as the identifier though, which was the other reason for preferring query params.